nova/core/roles/caddy/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Including Caddy installation tasks...
+  ansible.builtin.include_tasks: install.yml
+
+- name: Including Caddyfile with variables mode tasks...
+  ansible.builtin.include_tasks: caddyfile_with_vars.yml
+  when: caddy_template_caddyfile_with_vars
+
+- name: Including Caddyfile template tasks...
+  ansible.builtin.include_tasks: caddyfile_with_vars.yml
+  when: caddy_template_caddyfile
+
+- name: Composing Caddy for {{ inventory_hostname }}...
+  ansible.builtin.shell: docker compose -f {{ caddy_config_folder }}/docker-compose.yml up -d
```

Comment:
Is it not better to use `community.docker.docker_compose`?

Reply:
`community.docker.docker_compose` is using deprecated Docker API and no one has updated it. Currently the better way is to use shell that uses latest Docker Compose that comes with the compose plugin and is written in go.